### ISSUE TITLE
Validate that Images after deployment are correctly using DS builds

### ISSUE THREAD

**Issue by nehaberry** [red-hat-storage/ocs-ci#903]
Currently, even when we use Downstream process of deployment, we do  not have a mechanism to confirm if indeed DS images were used for creating the PODs.  This could result in overlooking or missing out on issues like[ [1]](https://bugzilla.redhat.com/show_bug.cgi?id=1762267)

[1] - https://bugzilla.redhat.com/show_bug.cgi?id=1762267

**Comment by vasukulkarni** [comment]
@mbukatov are you planning to fix your PR which can be used to fix this?

**Comment by petr-balogh** [comment]
@nehaberry @vasukulkarni @clacroix12 @vavuthu  we currently have the function which allows us to collect information about all images used from CSV but should we really validate those images? As the image names can change and also the image URL can be different for stage or production registry testing. So not sure how do we want to validate those images as QE just consuming what's defined in CSV and this can be different as I said for stage/production testing.  

I think we should just report all those images used in mail but not sure about additional validation.

I can imagine this kind of validation in OCS DS build pipeline from engineering but don't want to hardcode some image names here in ocs-ci.

@andrewschoen @zmc  any thoughts?

**Comment by vasukulkarni** [comment]
@petr-balogh yep there are too different builds, but I think we can add regex check to ensure no upstream images are in use (when downstream conf file is used) , that was one big image mix up issue.

**Comment by mbukatov** [comment]
We are no longer having such releng issues which would require this to be fully implemented.
